chrono-tz-build/Cargo.toml

```diff
@@ -6,16 +6,19 @@ rust-version = "1.60"
 description = "internal build script for chrono-tz"
 readme = "README.md"
 license = "MIT OR Apache-2.0"
+keywords = ["script", "chrono-tz", "timezone", "iana"]
+categories = ["development-tools::build-utils"]
 repository = "https://github.com/chronotope/chrono-tz"
 documentation = "https://docs.rs/chrono-tz-build"
 
 [features]
 filter-by-regex = ["regex"]
 case-insensitive = ["uncased", "phf/uncased"]
+regex = ["dep:regex"]
 
 [dependencies]
 parse-zoneinfo = { version = "0.3" }
-regex = { default-features = false, version = "1", optional = true }
+regex = { default-features = false, version = "1.0", optional = true }
```

Comment:
This is silly, IMO -- they're equivalent, and I prefer the previous form.

Reply:
It is. But adding two characters to get rid of a warning doesn't seem bad.

Reply:
GitHub works again.